src/middlewares/validations/reset-checkPassword.js

```diff
@@ -0,0 +1,22 @@
+import Joi from '@hapi/joi';
+
+const checkPasswordValidation = async (req, res, next) => {
+  const schema = await Joi.object({
+    password: Joi.string().min(8).required().messages({
+      'string.base': 'Password must be string',
+      'string.empty': 'Please fill in your Password',
+      'string.min': 'Password must be at least {#limit} characters long',
+      'any.required': 'Password is required'
+    }),
+    confirmPassword: Joi.string().min(8).required().messages({
+      'string.empty': 'Please confirm your Password',
+      'any.required': 'Confirm password is required'
+    })
```

Comment:
Validate **passwords** properly. Any 8 white space characters are passing.Add validation patterns

Reply:
I added REGEX to make it strong.